### DISCUSSION TITLE
Port to X-Plane 11

### DISCUSSION THREAD

**Discussion by pablodambrosio** [flybywiresim/aircraft#5313] [Ideas]
I'm not sure how hard it may be, but this is one of the best and more complete a320 I used, and it will be great to have it also available in x-plane. For sure Austin, the founder will be more than happy to help you, hi is a nice guy.

**Comment by hotshotp** [comment]
absolutely not, that would basically require rewriting all the systems, you have toliss and flight factor for the a320 mods

**Reply by FoxinTale** [reply]
While we as FBW won't port it ourselves, the license permits anyone to form their own group and port it themselves, as long as they keep the same license. Sure, Toliss and Flight Factor exist, but for most of us, that's far, far out of our price range. The average person (especially a hobbyist) doesn't always want (or literally cannot afford)  to invest more than the game/sim/thing itself for an addon.

**Reply by Paytheon** [reply]
Albeit, Toliss doesn't have auto-landing....LOL

**Comment by 2hwk** [comment]
It would require a big technical effort, as well as an additional support load. We already are stretched in multiple areas, and I do not feel that  the team will justify splitting resources to support this ourselves.
